halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/plugins/Plugin.java

```diff
@@ -37,6 +42,13 @@
   public Boolean enabled;
   public String manifestLocation;
 
+  @Setter(AccessLevel.NONE)
+  public HashMap<String, Object> options = new HashMap<>();
+
+  public void setOptions(HashMap<String, Object> options) {
+    this.options = generateOptions(options);
```

Comment:
Can't you remove that? You have a `@Setter` above. Also not sure why it has that `AccessLevel`